backend/src/LibExecutionStdLib/LibExecutionStdLib.fsproj

```diff
@@ -23,6 +23,7 @@
     <Compile Include="LibList.fs" />
     <Compile Include="LibMath.fs" />
     <Compile Include="LibNoModule.fs" />
+    <Compile Include="LibMaybe.fs" />
```

Comment:
This should be deleted, soon. I just think replacing Options/Results with a stdlib-defined enum should be a part of a separate PR, but wanted to have a demo like this for conversation.